fabric/src/main/java/dev/ftb/mods/ftbchunks/compat/commonprot/FTBChunksProtectionProvider.java

```diff
@@ -0,0 +1,146 @@
+package dev.ftb.mods.ftbchunks.compat.commonprot;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import com.mojang.authlib.GameProfile;
+import dev.ftb.mods.ftbchunks.FTBChunks;
+import dev.ftb.mods.ftbchunks.FTBChunksExpected;
+import dev.ftb.mods.ftbchunks.FTBChunksWorldConfig;
+import dev.ftb.mods.ftbchunks.data.ClaimedChunk;
+import dev.ftb.mods.ftbchunks.data.FTBChunksAPI;
+import dev.ftb.mods.ftbchunks.data.Protection;
+import dev.ftb.mods.ftblibrary.math.ChunkDimPos;
+import eu.pb4.common.protection.api.CommonProtection;
+import eu.pb4.common.protection.api.ProtectionProvider;
+import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents;
+import net.minecraft.core.BlockPos;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Explosion;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.phys.AABB;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.concurrent.TimeUnit;
+
+public class FTBChunksProtectionProvider implements ProtectionProvider {
+    public static ResourceLocation ID = new ResourceLocation(FTBChunks.MOD_ID, "provider");
+    private static LoadingCache<GameProfile, ServerPlayer> FAKE_PLAYERS = null;
+
+    public static void init() {
+        CommonProtection.register(ID, new FTBChunksProtectionProvider());
+
+        ServerLifecycleEvents.SERVER_STARTING.register(server -> {
+            FAKE_PLAYERS = CacheBuilder.newBuilder()
+                .initialCapacity(64)
+                .expireAfterWrite(
+                    30, TimeUnit.SECONDS)
+                .softValues()
+                .build(CacheLoader.from((profile) -> new ServerPlayer(server, server.overworld(), profile, null)));
```

Comment:
Could you create & use a class that extends `ServerPlayer` here (it doesn't need to add anything, just extend the class) ?

This will ensure that Architectury's `PlayerHooks.isFake()` method (which FTB Chunks uses) always returns true for these fake players (it checks  `player instanceof ServerPlayer && player.getClass() != ServerPlayer.class`)